src/iostream.c

```diff
@@ -387,7 +387,9 @@ void ChildStatusChanged( int whichsig )
           Pr("#E Unexpected waitpid error %d\n",errno, 0);
   } while (retcode != 0 && retcode != -1);
   
+#if !defined(LIBGAP)
```

Comment:
Could / should we replace this with a global switch, e.g.
```
  if (!disableGAPSignalHandler)
    signal(SIGCHLD, ChildStatusChanged);
```

This would make it easier to build both a GAP executable and a library without recompiling.